diff --git a/includes/uuid_term.features.inc b/includes/uuid_term.features.inc
index b055d45..10a223b 100644
--- a/includes/uuid_term.features.inc
+++ b/includes/uuid_term.features.inc
@@ -90,10 +90,10 @@ function uuid_term_features_export_render($module = 'foo', $data) {
     }
 
     // Export the parent uuids.
-    foreach ($terms as &amp;$term) {
-      if($parents = taxonomy_get_parents($term-&gt;tid)) {
+    foreach ($terms as &$term) {
+      if($parents = taxonomy_get_parents($term->tid)) {
         foreach ($parents as $parent) {
-          $term-&gt;parent[] = $parent-&gt;uuid;
+          $term->parent[] = $parent->uuid;
         }
       }
     }
diff --git a/uuid_features.module b/uuid_features.module
index f9a2464..c9c0176 100644
--- a/uuid_features.module
+++ b/uuid_features.module
@@ -61,10 +61,10 @@ function uuid_features_load_module_includes() {
  *
  * When exporting a vocabulary, include its terms.
  */
-function uuid_features_features_pipe_taxonomy_alter(&amp;$pipe, $data, $export) {
+function uuid_features_features_pipe_taxonomy_alter(&$pipe, $data, $export) {
   if ($vocab = taxonomy_vocabulary_machine_name_load($data)) {
-    foreach (taxonomy_get_tree($vocab-&gt;vid) as $term) {
-      uuid_term_features_get_dependencies($export, $term-&gt;uuid);
+    foreach (taxonomy_get_tree($vocab->vid) as $term) {
+      uuid_term_features_get_dependencies($export, $term->uuid);
     }
     $pipe['uuid_term'] = $export['features']['uuid_term'];
   }
